creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -28,6 +28,9 @@ const CLOCK_STATUS_REQUEST_TIMEOUT_MS = 2000 // 2s
 
 const MAX_USER_BATCH_CLOCK_FETCH_RETRIES = 5
 
+// Number of users to process in each batch for this._aggregateOpsWithQueriedSpIds
```

Comment:
now that we have removed the old withoutQueriedspIds, i think we can also rename this fn to indicate that it is simply aggregating op for a single user ie mentioning "withQueriedSpIDs" is no longer relevant/helpful

Reply:
for sure!